website/content/docs/concepts/mount-migration.mdx

```diff
@@ -0,0 +1,86 @@
+---
+layout: docs
+page_title: Mount Migration
+description: Move secrets and auth mounts within and across namespaces
+---
+
+# Mount Migration
+
+Vault supports moving secrets and auth mounts, both within and across namespaces. The operation
+is supported by API endpoints as well as a CLI helper around them.
+
+## Why
+
+There can be several reasons for wanting to migrate mounts. On OSS, the use cases could be for 
+renaming mounts in order to align with org standards. 
+On Enterprise, there are a few more potential use cases as namespaces come into the picture. 
+When migrating from an OSS binary to an Enterprise binary, organizations may want to divide their 
+mounts across several namespaces. Mounts may also be moved across namespaces when there is a reorganization
```

Comment:
```suggestion
mounts across several namespaces. Mounts may also be moved across namespaces when roles and responsibilities are necessary for reorganization.
```

Reply:
Hmm I think this sentence should stay as is, it is roles and responsibilities i.e some secrets engine owned by the core team is given to the devex team's namespace, that are being reorganized. wdyt?